src/main/java/org/folio/service/impl/DcbServiceImpl.java

```diff
@@ -36,27 +38,38 @@ public DcbServiceImpl(@Autowired DcbEcsTransactionClient dcbEcsTransactionClient
   }
 
   @Override
-  public void createTransactions(EcsTlrEntity ecsTlr) {
```

Comment:
Can we leave method `createTransactions` that will call `createLendingTransaction` and `createBorrowingTransaction`? We will have another place where we will want to create transactions soon, it would be nice to have a method that can create both transactions